server/controllers/documents/Show.js

```diff
@@ -0,0 +1,20 @@
+const Documents = require('../../models/Document')
+const Students = require('../../models/Student')
+
+module.exports = async function listUsers(ctx) {
+  const { studentID } = ctx.params
+  if (
+    isNaN(studentID) ||
```

Comment:
Creio que se for NaN ele não encontra o usuário e retorna 404, um erro já é suficiente.

Reply:
Faz sentido.
Manteve-se apenas a segunda verificação.